conda/models/dist.py

```diff
@@ -21,6 +21,9 @@
                                          'dist_name'))
 
 
+IndexRecord = PackageRecord  # for conda-build backward compat
```

Comment:
Is conda build accessing this module directly or should we expose it in the `api` module then?

Reply:
It should be in exports right now.  That's where conda-build should be accessing it from.  I'm not sure I want to expose it yet as something that can be created through the api.  Maybe; maybe not.

Reply:
Yeah exports module, so should we add this renaming there...?

Reply:
@kalefranz ?

Reply:
We can't for a while.

https://github.com/conda/conda-build/blob/3.5.1/conda_build/conda_interface.py#L67